libs/hooks/open-telemetry/README.md

```diff
@@ -1,21 +1,59 @@
 # OpenTelemetry Hook
 
+The OpenTelemetry hook for OpenFeature provides a [spec compliant][otel-spec] way to automatically add a feature flag evaluation to a span as a span event. Since feature flags are dynamic and affect runtime behavior, it’s important to collect relevant feature flag telemetry signals. This can be used to determine the impact a feature has on a request, enabling enhanced observability use cases, such as A/B testing or progressive feature releases.
+
 ## Installation
 
 ```
 $ npm install @openfeature/open-telemetry-hook
 ```
 
-Required peer dependencies
+### Peer dependencies
+
+Confirm that the following peer dependencies are installed.
 
 ```
 $ npm install @openfeature/js-sdk @opentelemetry/api
 ```
 
-## Building
+## Usage
+
+OpenFeature provider various ways to register hooks. The location that a hook is registered affects when the hook is run. It's recommended to register the `OpenTelemetryHook` globally in most situations but it's possible to only enable the hook on specific clients. You should **never** register the `OpenTelemetryHook` globally and on a client.
```

Comment:
We could probably implement a clever mechanism to detect if the Hook is run multiple times in an evaluation and warn, but it might not be worth it.

Reply:
I don't think it's worth it since it's a misconfiguration if you register any plugin at multiple layers.